### PR TITLE
Added UR5 robot descriptions with smaller table and without table

### PR DESCRIPTION
**Pull request by carlosquinterop**
- Added a new robot description with a smaller table (ur5_robotiq_robot_limited.urdf.xacro)
- Added a new robot description without table (ur5_robotiq_robot_limited_notable.urdf.xacro)
- Added a new robot semantic description without table (ur5_robotiq_notable.srdf)

### CODE REVIEW

**Review by ChamzasKonstantinos — APPROVED** [review]
Looks good

File: ur/config/ur5/ur5_robotiq85.srdf.xacro [hunk 1]
```diff
@@ -52,17 +52,15 @@
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="moveit_eef" parent_link="ee_link" group="gripper" />
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="fixed_joint" type="fixed" parent_frame="world" child_link="box_link" />
+    <!--virtual_joint name="fixed_joint" type="fixed" parent_frame="world" child_link="box_link" /-->
```
Comment (zkingston) [inline ur/config/ur5/ur5_robotiq85.srdf.xacro:55]: Can we just remove this line?

**Review by zkingston — APPROVED** [review]
Also, should there be a YAML configuration for the default UR5 without a table?

**Comment by ChamzasKonstantinos** [comment]
> Also, should there be a YAML configuration for the default UR5 without a table?

Yeah that already exists, it just did not show up because it did not change, (Now it should since I fixed the .xacro extension). 
Don't merge this yet, I want to run some more tests to make sure everything is in order

**Comment by ChamzasKonstantinos** [comment]
Performed several tests and it look ok

**Pull request merged**